docs/EXTENDED-ARCHIVES.md

```diff
@@ -171,75 +176,79 @@ You may declare the following colors, in hexadecimal format:
 
 The client decides how to use them and where, if at all.
 
-#### `image.name (String)`
+#### `images[].name (String)`
 
 The human-friendly name of the image.
 
-#### `image.version (String)`
+#### `images[].version (String)`
 
 The version of the image.
 
-#### `image.url (String)`
+#### `images[].url (String)`
 
 The main url of the image.
 
-#### `image.supportUrl (String)`
+#### `images[].supportUrl (String)`
 
 The url where users can get general support for this image.
 
 This could be a link to a forum, IRC room, troubleshooting page, support form,
 etc.
 
-#### `image.logo (String)`
+#### `images[].logo (String)`
 
 The path to a logo that represents the image, relative to the `.meta`
 directory.
 
-#### `image.releaseDate (String)`
+#### `images[].releaseDate (String)`
 
 The release date timestamp. The date should conform to [ISO 8601][iso8601]
 standard.
 
-#### `image.checksumType (String)`
+#### `images[].checksum.type (String)`
 
 The checksum type. The current possible values are: `sha1`, `sha256`, `crc32`,
 and `md5`.
 
-See the `.meta/checksum` file.
+#### `images[].checksum.path (String)`
 
-#### `image.path (String)`
+The path to a file containing the checksum, relative to the root of the
+archive. This file **can't be inside `.meta/`** and should be included **at the
+tail of the archive**.
```

Comment:
If we're including multiple images in an extended archive, I guess a more accurate restriction would be that the checksum has to come after it's associated image file, and not necessarily at the end of the archive.
I.e. you could have
```
.meta/manifest.json
<metadata files>
image1.img
image2.img
checksum1
checksum2
```
or
```
.meta/manifest.json
<metadata files>
image1.img
checksum1
image2.img
checksum2
```

For extendability purposes (e.g. there was a suggestion elsewhere about supporting multiple checksum types in the future), I'm beginning to wonder if the checksum should actually be a JSON file, rather than just flat-text?